xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1030 @@
+      </screen>
+    </step>
+    <step>
+     <para>
+      Replace the charts download URL in the index file for the charts
+      repository and rename the file back to <filename>index.yaml</filename>.
+     </para>
+     <screen>
+&prompt.user;<command>cd /tmp/charts</command>
+</screen><screen>
+&prompt.user;<command>sed -i -e 's/https:\/\/kubernetes-charts.suse.com/https:\/\/<replaceable>&lt;internal-mirror.hostname&gt;</replaceable>\/charts/g' \
+downloaded-index.yaml</command>
+</screen><screen>
+&prompt.user;<command>mv downloaded-index.yaml index.yaml</command>
+     </screen>
+    </step>
+   </procedure>
+
+    <procedure>
+     <title>Prepare And Transfer Data To Secure Storage</title>
+     <step>
+      <para>
+
+      </para>
+     </step>
+    </procedure>
+
+    <procedure>
+     <title>Update Internal Mirror</title>
+     <step>
+      <para>
+
+      </para>
+     </step>
+    </procedure>
+
+    <procedure>
+     <title>Refresh information on the &productname; cluster</title>
+     <step>
+      <para>
+       Update the repository information on the &productname; admin node.
+      </para>
+<screen>&prompt.user;<command>helm repo update</command>
+       </screen>
+     </step>
+    </procedure>
+
+    <para>
+     You can now deploy additional software on your &productname;. Refer to:
+     <xref linkend="sec.admin.software.install" />.
+    </para>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.caasp-deployment">
+   <title>Deploying &productname;</title>
+   <sect3>
+    <title>Using the ISO</title>
+    <para>
+     From YaST register the node against the RMT server. This will ensure the
+     final node zypper repositories are pointed against RMT, moreover all the
+     available updates are going to be installed (-> no need to run a
+     transactional-update right after the installation -> way faster).
+    </para>
+   </sect3>
+   <sect3>
+    <title>Using AutoYast</title>
+    <para>
+     Ensure the admin node is registered against RMT, that will ensure the
+     nodes that are provisioned by AutoYaST are registered against RMT to have all
+     the updates applied
+    </para>
+   </sect3>
+   <sect3>
+    <title>Using a prebuilt image (eg: KVM, Xen)</title>
+    <para>
+     The node has to be registered against RMT. This should be done in the same
+     way as a regular SLE machine: via SUSEConnect.
+    </para>
+   </sect3>
+   <sect3>
+    <title>Existing unregistered running node</title>
+    <para>
+     Again, they should be using SUSEConnect.
+    </para>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.troubleshooting">
+   <title>Troubleshooting</title>
+   <sect3>
+    <title>Skopeo Fails Because Of Self Signed Certificate</title>
+    <para>
+     If you are using a self-signed certificate for the registry you can use
+     <literal>--dest-tls-verify=false</literal> to skip the integrity check.
```

Comment:
Or add it to the system level certificates:

```
sudo cp <cert> /etc/pki/trust/anchors/<cert>
sudo update-ca-certificates
```

Reply:
@marcov I think skopeo also has a flag to load the certificates from a specific file (so without having to add them to the system ones)

Reply:
That's right, you need to have on the host a copy of the server certificate (i.e. the .crt file).

Then you can use the `--dest-cert-dir` flag instead of `--dest-tls-verify=false`, like this:

```bash
skopeo sync --dest-cert-dir /path/to/the/cert/dir docker://alpine:latest docker://localhost:5000
```

Reply:
Great, let's update this section of the documentation. I don't want to encourage bad security behaviours